internal/operators/manager_test.go

```diff
@@ -479,7 +479,7 @@ var _ = Describe("Operators manager", func() {
 			Expect(results).To(ContainElements(
 				api.ValidationResult{Status: api.Success, ValidationId: string(models.ClusterValidationIDLsoRequirementsSatisfied), Reasons: []string{}},
 				api.ValidationResult{Status: api.Failure, ValidationId: string(models.ClusterValidationIDOdfRequirementsSatisfied),
-					Reasons: []string{"A cluster with only masters or with a minimum of 3 workers is required."}},
+					Reasons: []string{"The cluster must either have no dedicated worker nodes or at least three. Add or remove hosts, or change their roles configurations to meet the requirement."}},
```

Comment:
NIT:
Should read
"The cluster must either have no dedicated worker nodes or at least three. Add or remove hosts, or change their role configurations to meet the requirement."

Reply:
I agree with you, but I wanted to use the same as in - https://github.com/openshift/assisted-service/blob/0e3133ea2cef252e63573178db88186e84b927e1/internal/cluster/validator.go#L362 (I didn't wrote that one, AFAIU it is UX.) Lets keep compatibility